packages/esm-service-queues-app/src/transition-latest-queue-entry/add-patient-to-queue-entry/add-patient-to-queue-entry.component.tsx

```diff
@@ -0,0 +1,72 @@
+import { Button, Form, ModalBody, ModalFooter, ModalHeader, Stack } from '@carbon/react';
+import { useVisit, type Visit } from '@openmrs/esm-framework';
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import QueueFields from '../../create-queue-entry/queue-fields/queue-fields.component';
+import { useMutateQueueEntries } from '../../hooks/useQueueEntries';
+import styles from './add-patient-to-queue-entry.scss';
+
+interface AddPatientToQueueEntryProps {
+  modalTitle: string;
+  patientUuid: string;
+  closeModal: () => void;
+}
+
+const AddPatientToQueueEntry: React.FC<AddPatientToQueueEntryProps> = ({ modalTitle, patientUuid, closeModal }) => {
+  const { t } = useTranslation();
+  const { activeVisit, isLoading: isLoadingVisit } = useVisit(patientUuid);
+
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const { mutateQueueEntries } = useMutateQueueEntries();
+  const [callback, setCallback] = useState<{
+    submitQueueEntry: (visit: Visit) => Promise<any>;
+  }>(null);
+
+  const handleSubmit = useCallback(
+    (event) => {
+      event.preventDefault();
+      setIsSubmitting(true);
+
+      callback
+        ?.submitQueueEntry?.(activeVisit)
+        ?.then(() => {
+          closeModal();
+          mutateQueueEntries();
+        })
+        ?.finally(() => {
+          setIsSubmitting(false);
+        });
+    },
+    [callback, activeVisit, closeModal, mutateQueueEntries],
+  );
+
+  if (isLoadingVisit) {
+    return null;
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
```

Comment:
The modal container has excessive horizontal padding:

![CleanShot 2025-03-10 at 12  14 29@2x](https://github.com/user-attachments/assets/b41d2a2f-b455-4728-bc4e-21cfdb7b0798)

We should constrain it, possibly by adding this styling to the parent `Form` component:

```scss
.container {
  :global {
    .cds--modal-content {
      padding: 0 !important;
    }
  }
} 
```